tests/functional/spec/features/last_viewed_guidance/last_viewed_guidance.spec.js

```diff
@@ -0,0 +1,56 @@
+const JwtHelper = require("../../../jwt_helper");
+
+const AddressConfirmationPage = require('../../../generated_pages/last_viewed_question_guidance/address-confirmation.page.js');
+const HouseholdInterstitialPage = require('../../../generated_pages/last_viewed_question_guidance/household-interstitial.page.js');
+const PrimaryPersonListCollectorPage = require('../../../generated_pages/last_viewed_question_guidance/primary-person-list-collector.page.js');
+
+describe('Last viewed question guidance', function () {
+
+  const last_viewed_question_text = 'This is the last viewed question in this section'
+  const last_viewed_question_url_arg='last_viewed_question_guidance=True'
```

Comment:
I don't think we should check the url arg in the functional tests. It is tested in the integration tests and doesn't add any value here.